rubocop.gemspec

```diff
@@ -38,6 +38,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('parallel', '~> 1.10')
   s.add_runtime_dependency('parser', '>= 2.5', '!= 2.5.1.1')
   s.add_runtime_dependency('powerpack', '~> 0.1')
+  s.add_runtime_dependency('psych', '>= 3.1.0')
```

Comment:
This change needs Psych v3.0.0 or higher to use `start_line` method.
But I choose to use `v3.1.0` or higher because we can simplify calling `Psych.load` method argument by using `filename` keyword argument since v3.1.0.
https://github.com/rubocop-hq/rubocop/pull/6733/files#diff-e2fc41e8fa0504d203570a8d822495afL187
So I think v3.1.0 is better.